app/models.py

```
@@ -0,0 +1,22 @@
+class ComparisonResponse(BaseModel):
+    status: str
+    message: str
+    diff_file: Optional[str] = None
+    timestamp: datetime = datetime.now()
+
+class ComparisonResult(BaseModel):
+    diff_content: str
+    repo_name: str
+    old_commit: str
+    new_commit: str
+    timestamp: datetime = datetime.now() 
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)